distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -75,7 +75,7 @@ grant codeBase "file:/platform-osgi-condpermadmin/acdebugger-backdoor" {
     permission org.osgi.framework.ServicePermission "org.codice.acdebugger.PermissionService", "get, register";
     permission org.osgi.framework.ServicePermission "org.osgi.service.condpermadmin.ConditionalPermissionAdmin", "get";
     permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
-    permission java.util.PropertyPermission "/", "read";
+    permission java.util.PropertyPermission "*", "read";
```

Comment:
What does this change do? Just curious.

Reply:
The backdoor is a bundle that is called directly by the AC debugger via the Java Debug Protocol. With this new version of the debugger, the backdoor now compresses the filenames reported by FilePermissions using system properties such that something like `/project/ddf-2.14.0-SNAPSHOT/etc/something/file.txt` is compressed to `${ddf.home.perm}etc${/}something${/}file.txt`. To achieve that, it needs accessed to all properties. The old version would only compress the file separator character to `${/}`

Reply:
It'd be great if we deployed a security policy file from the acdebugger repo and included in the distribution so we can avoid these type of changes over time.

Reply:
Maybe... maybe not.
This is like taking a blind policy file coming from a thirdparty bundle we include in our distro. I am not sure you want to do that blind.
Security shouldn't be blind and using some code that is not in a repo should force you to make a decision as to what you are willing to grant.

Reply:
I wonder if local bundle permissions would be better for the debugger.

https://osgi.org/specification/osgi.core/7.0.0/service.condpermadmin.html#i1576555

Reply:
This would require signing the debugger and updating our protection to only allow signed bundles to provide their own permissions. This is more involved than what we are willing to do for now especially since we are not sure if we will be keeping the security manager around in the future. If we wish to go this way, I would suggest we open a separate issue on the ac debugger directly to add that support first and then consider the changes involved in DDF to support that.